src/ZPublisher/HTTPResponse.py

```diff
@@ -433,10 +433,12 @@ def insertBase(self):
                     self.setHeader('content-length', len(self.body))
 
     def isHTML(self, text):
-        try:
-            text = text.decode(self.charset)
-        except UnicodeDecodeError:
-            pass
+        if isinstance(text, bytes):
+            try:
+                text = text.decode(self.charset)
+            except UnicodeDecodeError:
+                import pdb; pdb.set_trace()
```

Comment:
Oops, needs to ditch the `pdb`.

Reply:
Oops, sorry.

Reply:
Good that there is a the squash and merge feature